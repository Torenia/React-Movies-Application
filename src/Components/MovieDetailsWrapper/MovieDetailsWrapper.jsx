import styled from 'styled-components';

export const MovieDetailsWrapper = styled.div`
  height: 455px;
  background: rgb(67,67,67);
  background: linear-gradient(90deg, rgba(70,70,70,1) 1%, rgba(37,37,90,1) 54%, rgba(70,70,70,1) 100%);
  display: flex;
  justify-content: space-around;
  padding: 80px 60px 30px 60px;
  
  div {
    margin: 0 20px 0 0;
  }
`;
