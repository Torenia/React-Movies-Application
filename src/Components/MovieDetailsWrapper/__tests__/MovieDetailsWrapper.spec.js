import React from 'react';

import { MovieDetailsWrapper } from '../MovieDetailsWrapper';

describe('<MovieDetailsWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MovieDetailsWrapper />);
  });

  it('should render a movie details wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
