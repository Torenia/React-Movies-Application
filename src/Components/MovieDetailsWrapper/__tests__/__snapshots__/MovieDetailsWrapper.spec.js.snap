// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieDetailsWrapper /> should render a movie details wrapper 1`] = `
.c0 {
  height: 455px;
  background: rgb(67,67,67);
  background: linear-gradient(90deg,rgba(70,70,70,1) 1%,rgba(37,37,90,1) 54%,rgba(70,70,70,1) 100%);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  padding: 80px 60px 30px 60px;
}

.c0 div {
  margin: 0 20px 0 0;
}

<div
  className="c0"
/>
`;
